ocs_ci/ocs/utils.py

```diff
@@ -1003,6 +1048,41 @@ def collect_prometheus_metrics(
             json.dump(datapoints.json(), outfile)
 
 
+def save_live_logs(request, pods_containers_dict, patterns_to_log=None):
+    """
+    Save pods logs to files
+
+    """
+    tc_name = request.node.name
+
+    def finalizer():
+        """
+        Stop the thread that executed save_logs()
+
+        """
+        ocsci_config.RUN["save_logs"] = False
+        if thread:
+            thread.join()
+
+    request.addfinalizer(finalizer)
+
+    ocsci_config.RUN["save_logs"] = True
+
+    def save_logs():
+        while ocsci_config.RUN["save_logs"]:
+            time.sleep(10)
+            if ocsci_config.RUN["save_logs"]:
+                try:
+                    save_pods_logs_to_file(
+                        pods_containers_dict, tc_name, patterns_to_log
+                    )
+                except Exception:
```

Comment:
This sounds like:
https://realpython.com/the-most-diabolical-python-antipattern/

please at least write the exception to debug log level.

Reply:
:) 
Sure, will do